src/lib/services/graphql/mock-set-interval.ts

```diff
@@ -0,0 +1,57 @@
+/* global globalThis: false */
```

Comment:
`globalThis` is available in modern JS environments as an alias for the global object (window for the browser, global for Node...)